proposals/dao/vip_7.ts

```diff
@@ -0,0 +1,140 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner } from '@test/helpers';
+
+/*
+
+Timelock Proposal #7
+
+Description: Pauses minting on FEI PSM, remove VOLT liquidity from FEI PSM, adds DAI PSM to PCVGuardian whitelist, unpause redemptions on the USDC PSM
+
+Steps:
+  1 - Pause Minting on the FEI PSM
+  2 - Remove all VOLT from FEI PSM
+  3 - Add DAI PSM to whitelisted addresses on PCV Guardian
+  4 - Unpause redemptions for USDC PSM
+
+*/
+
+const vipNumber = '7';
+
+const voltFloorPrice = 9_000;
+const voltCeilingPrice = 10_000;
+
+const daiReservesThreshold = ethers.constants.MaxUint256;
+// these variables are currently unused as the PSM doesn't have the ability to mint
+const mintLimitPerSecond = ethers.utils.parseEther('10000');
+const voltPSMBufferCap = ethers.utils.parseEther('10000000');
+const addressOne = '0x0000000000000000000000000000000000000001';
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const priceBoundPSM = await (
```

Comment:
I would probably call this the `daiPSM` rather than `priceBoundPSM` to make it clear